m4/pyproject_toml_metadata.m4

```diff
@@ -17,5 +17,10 @@ classifiers = [
     "Programming Language :: Python :: Implementation :: CPython",
     "Topic :: Scientific/Engineering :: Mathematics",
 ]
-urls = {Homepage = "https://www.sagemath.org"}
+urls = {download = "https://doc.sagemath.org/html/en/installation/index.html",
+        "release notes" = "https://github.com/sagemath/sage/releases",
```

Comment:
Is "download" correct?

Reply:
The url names are capitalized?

https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#urls

Reply:
I've been following https://github.com/scipy/scipy/blob/main/pyproject.toml#L116 and some other projects. When displaying this info, PyPI automatically sentence-cases the URL names.

Reply:
I see.

"download" points to the installation guide. How about "installation"?

Reply:
> "download" points to the installation guide.

Yes, this is deliberate; same as on our website

Reply:
Hmm. OK.